managed/ui/src/components/certificates/Certificates.js

```diff
@@ -153,6 +173,14 @@ class Certificates extends Component {
         >
           <i className="fa fa-download"></i> Download Root CA Cert
         </MenuItem>
+        <MenuItem
+          onClick={() => { 
+            !deleteDisabled && this.deleteRootCertificate(payload?.uuid)
+          }}
+          disabled={deleteDisabled}
+        >
+          <i className="fa fa-trash"></i> Delete Root CA Cert
```

Comment:
missed this one - as long as it's a generic cert delete functionality `root CA` part is unnecessary

Reply:
Done